src/modules/esl-forms/esl-select/core/esl-select-renderer.ts

```diff
@@ -21,28 +21,41 @@ export class ESLSelectRenderer extends ESLBaseElement {
   /** Marker attribute to reflect filled state */
   @boolAttr() public hasValue: boolean;
 
-  protected $container: HTMLDivElement;
-  protected $rest: HTMLElement;
-  protected $text: HTMLElement;
-  protected $remove: HTMLButtonElement;
+  /** Internal container */
+  @memoize()
+  protected get $container(): HTMLElement {
+    const $container = document.createElement('div');
+    $container.classList.add('esl-select-text-container');
+    $container.appendChild(this.$text);
+    $container.appendChild(this.$rest);
+    return $container;
+  }
 
-  constructor() {
-    super();
+  /** Inner remove button */
+  @memoize()
+  protected get $remove(): HTMLElement {
+    const $remove = document.createElement('button');
+    $remove.type = 'button';
+    $remove.setAttribute('aria-label', 'Clear');
+    $remove.classList.add('esl-select-clear-btn');
+    $remove.classList.add('icon-nav-close-menu');
```

Comment:
```suggestion
    $remove.className = 'esl-select-clear-btn icon-nav-close-menu';
```
Its shorter. Besides, className is much faster.